crates/maybenot-ffi/README.md

```diff
@@ -0,0 +1,28 @@
+# Maybenot FFI
+
+This crate contains C FFI bindings for Maybenot, which let's you use Maybenot as a static library
+for languages other than Rust. Headers are found at `maybenot-ffi/maybenot.h` and are
+auto-generated when compiling using `make`.
+
+## Building
+You need to have [rust](https://rustup.rs/) installed.
+`cbindgen` is also required: `cargo install --force cbindgen`
+Then just run `make` to build a static library at `maybenot-ffi/libmaybenot.a`.
+
+Arguments to `make`:
+- `CARGO` override the `cargo` command
+- `TARGET` override target architecture; cross-compile.
+- `PROFILE` override the cargo profile, valid options are `release` and `debug`.
```

Comment:
Could be useful to point out what the default is.

Reply:
Done